src/pages/Student/Student.css

```diff
@@ -0,0 +1,201 @@
+.student__container{
+    max-width:1200px;
+    margin: auto;
+}
+  .student__main{
+      display:flex;
+      justify-content: space-between;
+     padding-top: 70px;
+  }
+  
+  .student__info {
+    padding-top: 100px;
+    align:left;
+  }
+  
+  .student__h1 {
+    width: 40rem;
+    font-size: 6rem;
+    color:#141639;
+  }
+  
+  .student_p {
+    font-size: 1.50rem;
+    margin: 2rem 0;
+    font-weight: bold;
+    color: #73747b;
+  }
+  
+  .teacher1--img {
+    width: 590px;
+    height:470px;
+  }
+
+
+  .button1{
+      color:white;
+      background-color:orangered;
+      text-decoration:none;
+      padding:15px;
+      border-radius:1rem;
+      font-size:rem;
+  }
+
+  .button1:hover{
+      color:white;
+      background-color:rgb(241, 130, 56);
+  }
+
+  .Aboutedu1--img{
```

Comment:
aboutedu1, aboutedu2 these number games just mess up our code and make it difficult to understand for 3rd person, so kindly change it as well.